curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/step-026.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-26
 
 The code for an ordered list (`ol`) is similar to an unordered list, but list items in an ordered list are numbered when displayed.
 
-After the final second section' last `h3` element, add an ordered list with these three list items: `flea treatment`, `thunder` and `other cats`.
+After the second `section`'s last `h3` element, add an ordered list with these three list items: `flea treatment`, `thunder` and `other cats`.
```

Comment:
Just to mention, we try to avoid the combination of code-block and non-code-block words as much as possible, because it is difficult for translators to translate:
> ...`section`'s ...

Instead, we should be going for:
> ... `section` element's ...

Reply:
Should I create a new pull request to change it as such?

Reply:
I saw now that you have already done that.